evap/contributor/templates/contributor_index.html

```diff
@@ -179,11 +179,11 @@
                                                         <span class="fas fa-pencil-alt"></span>
                                                     </a>
                                                 {% endif %}
+                                                <a href="{% url 'contributor:evaluation_preview' evaluation.id %}" class="btn btn-sm btn-light"
```

Comment:
This is now inside the `if user is editor or delegate` part. The preview view, however, has the decorator `@responsible_or_contributor_or_delegate_required` (see https://github.com/fsr-de/EvaP/blob/c0eada344ece2bc6bb9750502cb5f9b217bb3792/evap/contributor/views.py#L172) and also checks for that relationship between user and evaluation (whether the user `is_user_responsible_or_contributor_or_delegate`)
This doesn't really match, does it? When a user is contributor in that evaluation, even though they are not an editor, we still want them to get that link (and they should see the page), right?

Reply:
Uhh fancy inline preview. Seriously github, you got some nice stuff going on there xD

Reply:
changed that